docs/src/main/asciidoc/infinispan-client-guide.adoc

```diff
@@ -303,20 +364,12 @@ The Infinispan client supports both indexed and non indexed querying as long as
 properties of the proto schema.
 
 Query builds upon the proto definitions you can configure when setting up the `ProtoStreamMarshaller`.
-Make sure to configure this marshaller before attempting querying. Once this is completed
-you must add another optional depdency: `infinispan-query-dsl` to start using queries.
+Either method of Serialization above will automatically register the schema with the server at
+startup, meaning that you will automatically gain the ability to query objects store with
```

Comment:
```suggestion
startup, meaning that you will automatically gain the ability to query the objects store with
```
but I'm not sure what you mean. Is it `the objects store`, `the objects stored`?

Reply:
typo: supposed to be stored